yt/visualization/fits_image.py

```diff
@@ -880,6 +886,16 @@ def construct_image(ds, axis, data_source, center, image_res, width, length_unit
             frb = data_source.to_frb(width[0], (nx, ny), height=width[1])
         else:
             frb = data_source.to_frb(width[0], (nx, ny), center=center, height=width[1])
+    elif isinstance(data_source, ParticleAxisAlignedDummyDataSource):
+        bounds = (
+            center[0] - width[0] / 2,
+            center[0] + width[0] / 2,
+            center[1] - width[1] / 2,
+            center[1] + width[1] / 2,
+        )
+        frb = ParticleImageBuffer(
+            data_source, bounds, (nx, ny), periodic=np.all(ds.periodicity)
```

Comment:
Is this the same as `all(ds.periodicit)`?

Reply:
yes